python/pyspark/mllib/regression.py

```diff
@@ -54,6 +54,10 @@ def __repr__(self):
         return "LabeledPoint(%s, %s)" % (self.label, self.features)
 
 
+from pyspark.mllib.util import Saveable
+from pyspark.mllib.util import Loader
```

Comment:
Should it be `from pyspark.mllib.util import Saveable, Loader` and stay at the top of the file?